cli/src/commands/tui/app.rs

```diff
@@ -193,7 +199,7 @@ impl App {
             Some(Action::Quit) => AppReturn::Exit,
             // TODO: just for prototype purpose
             // we will need to track current selected panel to apply the event
-            None => match key {
+            None | Some(Action::ToggleLogsTailing) => match key {
```

Comment:
```suggestion
            Some(Action::ToggleLogsTailing) => { ... }
            None => match key {
```
we can just match the `Action` instead